src/pages/Selector/transform-employees.js

```diff
@@ -0,0 +1,20 @@
+export const transformEmployees = employees => {
+  const transformedEmployees = employees.map(value => {
+    return { id: value.id, label: `${value.lastName}, ${value.firstName}` };
+  });
+
+  // sorting employees in alphabetical order
+  return transformedEmployees.sort((x, y) => {
```

Comment:
This could be extracted out to a sorting utility, take in an accessor.

Something like:
```
return sortCollection(transformedEmployees, x => x.label);
```